docs/pbc/all/payment-service-provider/202404.0/base-shop/third-party-integrations/payone/payone.md

```diff
@@ -2,10 +2,16 @@
 title: Integrate PayOne
 description: Payone partner integ
 template: concept-topic-template
-last_updated: Nov 21, 2023
+last_updated: Jul 31, 2024
 ---
 
 The Payone partner integration is available through the following integration methods:
 
+## Current limitations
+
+- Payments can be properly canceled only from the the Back Office and not from the Payone PMI.
+- Payments can't be partially canceled. We create one payment intent per order and it can either be authorized or fully cancelled.
+- When you cancel an item on the order details page, all order items are canceled.
+
```

Comment:
the section is inside another section, please move to bottom 
![image](https://github.com/user-attachments/assets/fdea91c4-7f10-4ab8-9943-5f4096bf3829)


Or to a specific page with the integration type `payone-integration-in-the-back-office`. Are you sure it's not supported by by eco module (manual integration type)?

Reply:
This question is for ECO team. but I guess - yes. only if for payone will be used specific OMS.